### PR TITLE
Astra strings audit

### PR DESCRIPTION
**Pull request by explorigin**
- Move server-side files from `translations` to `backend`.
- Lots of translation string cleanups

### CODE REVIEW

File: astra/backend/en/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+msgstr "Please check your credentials and try again."
+
+#: /application/astra/exceptions.py:56
+msgid "banned_account_msg"
+msgstr "Account temporarily locked: Too many failed login attempts. Your IP address is: %(addr)s"
+
+#: /application/astra/exceptions.py:57
+msgid "banned_account_desc"
+msgstr "Please contact support to unlock your account or try again shortly."
+
```
Comment (usethecodeluke) [inline astra/backend/en/LC_MESSAGES/messages.po]: Why the lack of consistency?  Should msgid match msgstr or no?

File: astra/backend/en/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+msgid "Your account information could not be retrieved, please try again or contact support."
+msgstr ""
+
+#: /application/astra/exceptions.py:90
+msgid "The current password for this user has expired."
+msgstr ""
+
+#: /application/astra/exceptions.py:91
+msgid "You must update your password before login."
+msgstr ""
```
Comment (usethecodeluke) [inline astra/backend/en/LC_MESSAGES/messages.po]: why is msgstr not populated for some?

File: astra/backend/ja/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -0,0 +1,369 @@
+# Japanese translations for Astra.
```
Comment (usethecodeluke) [inline astra/backend/ja/LC_MESSAGES/messages.po]: if we are staging for submission to smartling, should we remove our phony translations?

File: astra/backend/en/LC_MESSAGES/messages.po (hunk 1 above)
Comment (explorigin) [reply astra/backend/en/LC_MESSAGES/messages.po]: Some messages are longer and I wanted to pull them out of the code. The difference is inconsequential (just inconsistent).

File: astra/backend/en/LC_MESSAGES/messages.po (hunk 2 above)
Comment (explorigin) [reply astra/backend/en/LC_MESSAGES/messages.po]: In these cases, the _msgid_ falls through when _msgstr_ is empty, but for cleanliness, I'll move them down.

File: astra/backend/ja/LC_MESSAGES/messages.po (hunk 3 above)
Comment (explorigin) [reply astra/backend/ja/LC_MESSAGES/messages.po]: Yes

File: astra/backend/en/LC_MESSAGES/messages.po (hunk 2 above)
Comment (explorigin) [reply astra/backend/en/LC_MESSAGES/messages.po]: Fixed

File: astra/backend/en/LC_MESSAGES/messages.po (hunk 1 above)
Comment (explorigin) [reply astra/backend/en/LC_MESSAGES/messages.po]: Fixed

File: astra/backend/ja/LC_MESSAGES/messages.po (hunk 3 above)
Comment (explorigin) [reply astra/backend/ja/LC_MESSAGES/messages.po]: Fixed

**Pull request merged**